AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/EventAdminController.cs

```diff
@@ -185,6 +187,11 @@ private string GenerateItineraryDetailsTemplateUrl()
                         ModelState.AddModelError("ImageUrl", "You must upload a valid image file for the logo (.jpg, .png, .gif)");
                         return View(campaignEvent);
                     }
+
+                    if (existingImageUrl != null)
+                    {
+                        await _imageService.DeleteImageAsync(existingImageUrl);
+                    }
                 }
```

Comment:
Same comment as above regarding only deleting the existing image if we maybe had a success in the imageService during the new image upload (and get a new imageUrl back)